mail/gophish/patches/gophish-0.12.1-working-directory.patch

```diff
@@ -0,0 +1,27 @@
+--- a/gophish.go
```

Comment:
Is this backport or not? If it is backport, then it should have Git header.
If it is not backported, then please add Git header, which contains commit header, description, who is the author, etc and most likely try to send it to the upstream first.